config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -375,46 +375,3 @@ presubmits:
             memory: "9000Mi"
             # during the tests more like 3-20m is used
             cpu: 2000m
-  # conformance test against kubernetes release-1.15 branch with `kind`, skipping
```

Comment:
this has been somewhat useful for us, but we do support back to 1.13 still without CI for 1.13/1.14 ..

Reply:
I don't think we should force projects other than the main repo to follow the same lifecycle.

Reply:
I can see reason for a project like KinD to want to insure current KinD is able to continue to host older content (eg: 2018 Kubernetes on 2020 KinD).  So for "we" being "KinD", and "support" being "patches to KinD" there's sense in this.

I've dropped that patch and will look to not do this self-opinionated cleaning on kubernetes-sigs yaml vs just kubernetes core's.

Reply:
To be fair: with my test-infra cap on, I can also see a reason to institute some sort of policy that we want to restrict this, so as not to host kubernetes 1.1 testing or something, but I don't think that's yet been discussed, versus avoiding anything that looks like / costs like the main repo / core project continuing to support old versions that we're not in fact supporting.